indra/newview/pipeline.cpp

```diff
@@ -7476,7 +7478,7 @@ void LLPipeline::generateSMAABuffers(LLRenderTarget* src)
             LLGLSLShader& blend_weights_shader = gSMAABlendWeightsProgram[fsaa_quality];
 
             dest.bindTarget();
-            dest.clear(GL_COLOR_BUFFER_BIT);
+            dest.invalidate(GL_COLOR_BUFFER_BIT);
```

Comment:
This could have side effects on SMAA as the edge detection shaders utilize discard.

Reply:
That's actually a pretty good point - discards get weird when you don't have something clear.  Do you have any good scenes to test with?